### ISSUE TITLE
Proper way to delete an HomeServer?

### ISSUE THREAD

**Issue by ArchangeGabriel** [matrix-org/matrix-doc#2331]
I’m in the process of deleting my HomeServer (because I’ve migrated to another hosted elsewhere). I’ve made the only user, me leave all rooms it was part of, but I still see a lot of incoming traffic for Matrix (while my HS is currently down, so I’m looking for that at nginx level).

Is there some proper way of advertising deletion of an HS to the federated network?

**Comment by turt2live** [comment]
There's not currently a way to advertise the deletion (and I'm failing to find the issue that would add it), though the traffic you're seeing is probably safe to just end. Most of the queries coming in are probably for keys and events you've sent, though this traffic isn't critical to the operation of a homeserver: homeservers should be resilient enough that when another server disappears it can still get the information it needs. For instance, it can get events sent by your server through another server and keys can come from a notary server (a server that has cached your public signing key).

In short: should be able to just turn the thing off, with or without leaving all your rooms. 

Awkwardly, it is best to ask these sorts of questions in [#matrix:matrix.org](https://matrix.to/#/#matrix:matrix.org) rather than here. https://github.com/matrix-org/matrix-doc/issues/831 is probably the issue that would solve this sort of thing going forward.
